docs/ru/1.guide/5.custom-theme.md

```
@@ -0,0 +1,571 @@
+
+Большинству сайтов документации достаточно Markdown и конфига. Но когда нужен
+свой лендинг, компонент в Markdown или новый layout страницы, undocs позволяет
+проекту документации поставлять свой Vue. Положите каталог `.docs/` рядом с
+Markdown — компоненты, страницы и layouts подхватятся работающим приложением.
+
+## Каталог `.docs/`
+
+Создайте папку `.docs/` рядом с Markdown. undocs находит три особых
+подкаталога:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)